documentation/swagger/index.html

```diff
@@ -1,602 +0,0 @@
-<!-- HTML for static distribution bundle build -->
```

Comment:
Why is this file changed / deleted?

Reply:
It's a generated file. I'm not sure why it would have changed.

Reply:
I'm going to build your branch locally. We are separately responding to an issue about this file but haven't done anything yet so this is oddly coincidental.

Reply:
I'm going to restore the file.